openverse_catalog/dags/providers/provider_api_scripts/flickr.py

```diff
@@ -126,6 +126,11 @@ def ingest_records(self, **kwargs):
                 )
         logger.info("Completed large batch processing by license type.")
 
+        # Report the final number of requests made. Because the Flickr DAG recieves a
```

Comment:
```suggestion
        # Report the final number of requests made. Because the Flickr DAG receives a
```
PyCharm is complaining